pkg/server/pod_controller.go

```diff
@@ -491,6 +449,29 @@ func (c *PodController) dispatchPodToNode(pod *api.Pod, node *api.Node) {
 	}
 }
 
+func (c *PodController) addCloudRoute(node *api.Node, cidrs []string) error {
+	instanceID := node.Status.InstanceID
+	err := c.cloudClient.ModifySourceDestinationCheck(instanceID, false)
+	if err != nil {
+		return err
+	}
+	for _, cidr := range cidrs {
+		if _, _, err := net.ParseCIDR(cidr); err != nil {
+			return err
+		}
+		if err := c.cloudClient.AddRoute(cidr, instanceID); err != nil {
```

Comment:
GarbageController doesn't clean up routes yet.  Should that get added?

Reply:
Adding it.